scripts/Reader.py

```diff
@@ -39,7 +39,7 @@ def __init__(self):
             sys.exit('Please run RegisterDevice.py first')
         else:
             with open(path + '/deviceName.txt', 'r') as f:
-                deviceName = f.read()
+                deviceName = f.read().rstrip()
```

Comment:
Should we trim left and right side?

Reply:
I currently don't see a big potential for errors from an untrimmed left side. I'd recommend to fix this only if we actually encounter it in the wild.